python/llm/src/bigdl/llm/optimize.py

```diff
@@ -133,6 +133,31 @@ def low_memory_init():
 
 
 def load_low_bit(model, model_path):
+    """
+    Load the optimized pytorch model.
+
+    :param model: The PyTorch model instance
+    :param model_path: The path of saved optimized model
+
+    return: The optimized model.
```

Comment:
return: The optimized model. => :return: The optimized model. 

Same for the other places in this page with return field